project/MimaBuild.scala

```diff
@@ -88,15 +88,7 @@ object MimaBuild {
 
   def mimaSettings(sparkHome: File, projectRef: ProjectRef) = {
     val organization = "org.apache.spark"
-    val previousSparkVersion = "1.6.0"
-    // This check can be removed post-2.0
-    val project = if (previousSparkVersion == "1.6.0" &&
-      projectRef.project == "streaming-kafka-0-8"
-    ) {
-      "streaming-kafka"
-    } else {
-      projectRef.project
-    }
+    val previousSparkVersion = "2.0.0"
```

Comment:
Isn't there a nightly repo that could be used for this? I generally think it's a good idea to switch versions as soon as possible, otherwise the person doing it later becomes responsible for figuring out mima failures...